far_mar.rb

```diff
@@ -0,0 +1,15 @@
+# FarMar/far_mar.rb
+
+# gems your project needs
+require 'csv'
+
+# ...require all needed classes
+require 'lib/markets_class.rb'
```

Comment:
@yaelittle I think these should be require_relative and should be after the module definition.